.github/workflows/release-apps.yml

```diff
@@ -0,0 +1,74 @@
+name: Release Apps
+
+on:
+  release:
+    types: [ published ]
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
```

Comment:
For security hardening should add permission.

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs
https://docs.github.com/en/rest/reference/permissions-required-for-github-apps

Reply:
I have been wondering what sort of permissions we may want to add here.

I assume we add permissions so that any third party package we are using is not able to perform operations on our repository they are not permitted to?

If my assumption is correct I will work to figure out what the minimum permissions are to fulfill our workflow.

Reply:
To create another PR for this, since it's low priority.

Reply:
Created a new ticket to track it: https://github.com/DeFiCh/jellyfish/issues/1151